.github/pull_request_template.md

```diff
@@ -0,0 +1,45 @@
+# Pull Request Template
+
+## 📄 Description
+
+<!--
+Provide a concise description of the changes introduced by this PR.
+-->
+
+## 📝 Changelog
+
+<!--
+**Required**: List all changes using Conventional Commit syntax.
+Each entry should start with a type, followed by a colon and a brief description.
+These entries will be scraped and included in the release tags.
+
+**Example:**
+- `feat: add user authentication module`
+- `fix: resolve login issue with special characters`
+- `docs: update API usage documentation`
+- `perf: improve performance of the login endpoint`
+- `refactor: refactor the authentication module`
+- `test: add unit tests for the authentication module`
+- `chore: update dependencies`
+
+Major changes should be marked with `!` and have a footer the `BREAKING CHANGE:` keyword.
```

Comment:
could you provide a more clear example of how that footer is supposed to be structured?